Course_Part_06/src/Employee.java

```diff
@@ -0,0 +1,36 @@
+public class Employee {
+
+    int id;
+    String firstName;
+    String lastName;
+    int salary;
+
+    Employee(int id, String firstName, String lastName, int salary){
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.salary = salary;
+    }
+
+    public int getAnnualSalary() {
+        return this.salary * 12;
+    }
+
+    public void printAnnualSalary(){
+        System.out.println("Annual salary is: " + this.getAnnualSalary());
+    }
+
+    public void raiseSalary(int percent){
+        this.salary = this.salary * (percent+100) / 100;
```

Comment:
CTRL + ALT + L voor uitlijning.

Onnodig gebruik van keyword 'this' (kijk even goed wanneer je het wel en niet gebruikt).

Prima implementatie om het salaris te verhogen, het kan ook op een andere manier:
`salary += salary * 0.01 * percent;`